### DISCUSSION TITLE
[RELEASE] Isolation

### DISCUSSION THREAD

**Discussion by TheLungy** [nzp-team/nzportable#882] [Map Releases]
Hey all, this is my first map titled Isolation. 

**Features:**
  _Teleporter/Zapper Trap/PAP_
  _Buyable ending (Jeep in garage)_
Easter Eggs

<details>
  <summary>Easter Eggs Spoilers!</summary>

**1. There are three teddy bears scattered around the map. When all three have been shot, a hidden room will reveal itself.**

<details>
  <summary>Locations:</summary>

1. Outside spawn room windows, on brick wall.
2. Pathway between the garage and factory/power/teleporter, look up on the dirt path.
3. Behind deadshot daiquiri, up on a tree branch.

Once all bears have been shot, will open a room in the factory which will allow you to wall-buy a PAP'd Wunderwaffe (4000).
</details>

**2. There are two hidden switches, when interacted with or shot, will reveal a hidden room. These two rooms are on a timer.** 

<details>
  <summary>Locations:</summary>

1. First door you purchase from spawn, you will see a small red button on the right hand side. If you shoot this button, it will reveal a PAP'd Double Barrel Shotgun (2000).
2. The PPSH wallbuy has a red button on the side. Press this button and go to the near small building that has MuleKick. This will reveal a PAP'd Raygun (3000).

</details>

**3. Lastly, there is a button when interacted with will reveal a teddy bear. Find the teddy bear, shoot it, and be treated to a song.**

<details>
  <summary>Locations:</summary>
The button is the 2nd story factory balcony. Activate this button, and in the garage with the jeep where zombies fall down - you will notice the Minecraft mossy cobblestone is gone, revealing a bear. Shoot this bear to play "beyond_me.ogg".
</details>`

</details>

My personal best is round 30 and decided to leave via the purchasable ending. There are a couple rooms where you can run trains/farm the zombies, however between the rooms are quite narrow. Be careful, especially with getting downed mid-round and having to get your perks again.

Let me know what issues you all experience in the discussions below.

[DOWNLOAD](https://archive.org/download/nzp-isolation_20240824_0250/NZP%20Isolation.zip)

A good strategy I found is running a train of zombies in the factory area, going upstairs, unloading a full mag, and then PhD flopping down back to ground level and running to the garage to activate the trap. This lasts until round 25 before zombies stop getting one-shot by PhD Flopper.

![screenshot 4](https://github.com/user-attachments/assets/092978b9-500e-41cf-8c25-2cb7c0003d2c)

![screenshot 3](https://github.com/user-attachments/assets/1e2852d8-5e0c-42fe-a193-f3c2a814e50d)

![screenshot 5](https://github.com/user-attachments/assets/b61644e7-e71a-4bfc-91c1-cb4908c8b83f)

![screenshot 6](https://github.com/user-attachments/assets/49fdada8-4c8f-43fd-a7e6-056e352e5dd7)

![Aerial View:](https://github.com/user-attachments/assets/389efaba-79fa-40a8-8e9a-56dcba2010c2)

![screenshot 2](https://github.com/user-attachments/assets/c921502c-1bb0-45e4-a9fb-64d369b89895)

<details>
  <summary>Change Log</summary>

8/23/24

1. Added Zapper Trap
2. Added Teleporter
3. Added PAP room, with Teleporter (Kino style, link once then use)
4. Recreated waypoints to include special waypoints. These are placed in the mesh of func_nzp_door, that way zombies will need to go the long way around, instead of getting stuck on the door since the waypoints tell them that's the quickest path to player.
5. Fixed Waypoints near Juggernaut
6. Removed tiny ground bumps causing zombies to randomly get stuck
7. Added a couple zombie spawns that "fall" into the map from the roof.
8. Updated secrets - secrets are now working as expected.
9. Converted fences to func_illusionary, fences are now able to be shot through.
10. Added "NoClip" green texture for fences, as zombies would be able to see direct LOS otherwise. (If zombies have LOS, they ignore waypoints - need both from eye level, and kneecap level)
11. Tinkered with railing in factory. So have no idea how to make that look nice.
12. Tinkered with teleporter stairs, zombies will still "fall" for a second when going down the steps every 5-6 steps. 
13. Added Zombie spawns in teleporter room to discourage camping
14. Misc cleanup and texturing. 
15. Fixed window zombie spawn in factory under 2nd story stairs area. (Zombies were rising from the ground, not where they were placed, and running from inside the map to the window and getting stuck at the window/2nd story, eventually dying from inactivity)

</details>

NZ:P Team for the textures.

**Comment by ToonLink0001** [comment]
Yo this map looks good! Keep up the good work!

**Comment by Stormshadow145** [comment]
has anyone tested if this works on PSP? I really wanna try it out but I'm busy at the moment

**Comment by MisaelJama** [comment]
Could you make it for PSP please?

**Comment by MotoLegacy** [comment]
This map has a compatibility issue with other platforms, that being fog. I realized there was not any documentation previously on how fog is supposed to work in NZ:P so I wrote a page: https://docs.nzp.gay/mapping/fog

I've also updated Vril engine to not allow any "bad" fog values to actually make it to render, as a mitigation against the issues seen here. The map should now be playable on PSP/3DS as a result, but please consider actually fixing your fog values!

**Comment by MotoLegacy** [comment]
Tagging as Meets Install Standard!

**Comment by Jeyphr** [comment]
Watch my review here: https://youtu.be/ZS4VN2ed-X0
( When it comes out )

**Reply by Jeyphr** [reply]
It's out now!

**Comment by V1cboi123** [comment]
Pretty good map.

**Comment by nonMat** [comment]
Great Atmosphere!

**Comment by Aduki-Sanchez** [comment]
I really love this map, is my favorite custom map so far, unfortunately I experimented an issue when I was playing on it, I was in that structure because the crate moved to it, then I tried to slide out of it because I was delaying the round and the zombie was cornering me, then I got trapped here
![Uploading Captura de pantalla de 2026-08-07 20-45-52.png…]()

**Reply by Aduki-Sanchez** [reply]
idk if my screenshot actually uploaded, but I got stuck in the closed spawn infront of the shed near the deadshot perk

**Comment by SillyZombie115** [comment]
I'm having some sort of compatability issue with this map, for PSP.
(PPSSPP emulator on Android phone)
When starting the game, the loading bar on the
loading-screen loads half-way and then gets stuck.
??
Gave it a full 10min to load, it was just completely stuck there.
Couldn't load into a game.

If anyone knows what the issue is, lmk.

**Reply by MotoLegacy** [reply]
File an NZP bug report.
